astropy/coordinates/sky_coordinate.py

```diff
@@ -172,6 +173,15 @@ def __init__(self, *args, **kwargs):
             raise ValueError('Cannot create a SkyCoord without data')
 
     @property
+    def info(self):
+        """
+        Mixin column information (attributes)
+        """
+        if not hasattr(self, '_info'):
+            self._info = ColumnInfo()
+        return self._info
```

Comment:
@embray - yup, the `@lazyproperty` looks nice here.

@mhvk - I've also been a little worried about `info` as being too generic.  `attrs` is probably less common (and also less informative in code).  Anyway `col_info` would be OK by me, though you end up with a lot of `col.col_info`.  Other suggestions?